.github/workflows/build-beta.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          fetch-depth: 500
+          fetch-depth: 0
```

Comment:
this prompts the action to also fetch tags, which was apparently dropped in v3 in favor of `fetch-tags: true`. We could add that flag, but eh.